src/internal/promutil/promutil.go

```diff
@@ -43,7 +43,7 @@ func (rt *loggingRT) RoundTrip(req *http.Request) (*http.Response, error) {
 	start := time.Now()
 	ctx := pctx.Child(req.Context(), "outgoingHttp", pctx.WithFields([]log.Field{
 		zap.String("name", rt.name),
-		zap.String("method", req.Method),
+		zap.String("http_method", req.Method),
```

Comment:
This is a good idea, _but_ does anything currently expect the HTTP method in a log field named `method`?